### PR TITLE
Add motus/merge and biom support

### LINKED ISSUE #99: Add mOTUs/merge
### Description of feature

Add mOTUs/merge module

### PR DESCRIPTION
**Pull request by jfy133**
Closes #99 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/taxprofiler/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/taxprofiler _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/test_motus.config [hunk 1]
```diff
@@ -38,4 +38,5 @@ params {
     run_centrifuge                        = false
     run_diamond                           = false
     run_motus                             = true
+    run_profile_standardisation           = true
```
[the remark this answers is not in the document]
Comment (jfy133) [reply conf/test_motus.config:41]: To activate the subworkflow, see line 227 of taxprofiler.nf.

File: workflows/taxprofiler.nf [hunk 2]
```diff
@@ -220,6 +221,14 @@ workflow TAXPROFILER {
         ch_versions = ch_versions.mix( VISUALIZATION_KRONA.out.versions )
     }
 
+    /*
+        SUBWORKFLOW: PROFILING STANDARDISATION
+    */
+    if ( params.run_profile_standardisation ) {
+        STANDARDISATION_PROFILES ( PROFILING.out.classifications, PROFILING.out.profiles, DB_CHECK.out.dbs, PROFILING.out.motu_version )
```
Comment (jfy133) [inline workflows/taxprofiler.nf:228]:
```suggestion
        STANDARDISATION_PROFILES ( PROFILING.out.classifications, PROFILING.out.profiles, DB_CHECK.out.dbs, PROFILING.out.motus_version )
```

File: subworkflows/local/profiling.nf [hunk 3]
```diff
@@ -227,12 +227,12 @@ workflow PROFILING {
...
         ch_versions        = ch_versions.mix( MOTUS_PROFILE.out.versions.first() )
         ch_raw_profiles    = ch_raw_profiles.mix( MOTUS_PROFILE.out.out )
-
     }
 
     emit:
     classifications = ch_raw_classifications
     profiles        = ch_raw_profiles    // channel: [ val(meta), [ reads ] ] - should be text files or biom
     versions        = ch_versions          // channel: [ versions.yml ]
+    motu_version    = params.run_motus ? MOTUS_PROFILE.out.versions.first() : Channel.empty()
```
Comment (jfy133) [inline subworkflows/local/profiling.nf:236]:
```suggestion
    motus_version    = params.run_motus ? MOTUS_PROFILE.out.versions.first() : Channel.empty()
```

File: conf/test_motus.config (hunk 1 above)
[the remark this answers is not in the document]
Comment (sofstam) [reply conf/test_motus.config:41]: General question: shall we have this parameter in the rest of configs as well? (`test.config`, `test_full.config`)

[the remark this answers is not in the document]
Comment (jfy133) [reply conf/test_motus.config:41]: In the future yes, once there is functionality for the other tools :)

[the remark this answers is not in the document]
Comment (sofstam) [reply conf/test_motus.config:41]: I can create an issue if needed.

**Review by sofstam — APPROVED** [review]

File: conf/test_motus.config (hunk 1 above)
[the remark this answers is not in the document]
Comment (jfy133) [reply conf/test_motus.config:41]: I think it's fine for now! I will be implied :)

**Pull request merged**